docs/performance/inserts/parallel.txt

```diff
@@ -0,0 +1,131 @@
+.. _parallel_inserts:
+
+================
+Parallel Inserts
+================
+
+Because CrateDB is a clustered data with every node capable of handling client
+requests, parallelizing (i.e. doing many at once) your inserts is one good way
```

Comment:
What would be another way?

Reply:
well, we're half way through a second in the docs called "inserts performance" so I am presuming that from context it's clear that this is one of multiple approaches being suggested

thoughts?